### PR TITLE
DT-1442: ⬆️🚧 Beta version to upgrade to application insights agent v3…

### PR DESCRIPTION
**Pull request by petergphillips**
… (beta)

### CODE REVIEW

**Review by mikehalmamoj — APPROVED** [review]

**Comment by mikehalmamoj** [comment]
Good readme update, but would be better if we could link to some examples (once we have some!) to make life easier when people make the switch.

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ import org.gradle.api.tasks.Copy
 import uk.gov.justice.digital.hmpps.gradle.ConfigManager
 
 private const val APP_INSIGHTS_SDK_VERSION = "2.6.2"
-private const val APP_INSIGHTS_AGENT_VERSION = "2.6.2"
+private const val APP_INSIGHTS_AGENT_VERSION = "3.0.1-BETA.2"
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt:8]: Is this what actually does the upgrade???

Comment (petergphillips) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt:8]: yes, we specify the new version of the agent here

**Pull request merged**